packages/xo-server/src/xapi/index.js

```diff
@@ -1156,7 +1157,7 @@ export default class Xapi extends XapiBase {
         vdis,
         vifsMap,
         {
-          force: 'true',
+          force: String(force),
```

Comment:
Is it how this is usually done? I'd say `force ? 'true' : 'false'` is safer and clearer.